sailthru_content/services/tests/catalog_api_test_mixins.py

```diff
@@ -16,7 +17,17 @@ def setUp(self):
         self.api_url_root = 'http://' + faker.domain_name()
         self.catalog_api_service = CatalogApiService(faker.password(), self.oauth_host, None, None, self.api_url_root)
 
-    def prepare_get_courses(self):
+    def prepare_get_courses(self, seat_type=None):
+        if seat_type:
+            seat = {
+                "type": seat_type,
```

Comment:
We are using single quotes throughout this file. You should use single quotes here.